src/aui-video/js/aui-video.js

```diff
@@ -486,14 +504,14 @@ var Video = A.Component.create({
          */
         _setResponsiveDimensions: function() {
             var aspectRatio,
-                currentTargetHeight,
-                currentTargetWidth,
-                instance,
-                height,
-                updatedHeight,
-                updatedWidth,
-                width,
-                winNode;
+            currentTargetHeight,
+            currentTargetWidth,
+            height,
+            instance,
+            updateHeight,
```

Comment:
This accidentally changed the name of the variable from `updatedHeight` to `updateHeight`, making the usage below create global variables.

I'll fix this on my side, but next time remember to run `gulp lint` before sending the pr, it catches problems like that (that's how I caught it) :)